php-classes/Slate/CBL/Tasks/StudentTask.php

```diff
@@ -188,7 +187,7 @@ public function getTaskSkills()
 
         if ($this->Skills) {
             foreach ($this->Skills as $skill) {
-                $currentLevel = $skill->Competency ? $skill->Competency->getCurrentLevelForStudent($this->Student) : null;
+                $currentLevel = $skill->Competency && $this->Student instanceof \Slate\People\Student ? $skill->Competency->getCurrentLevelForStudent($this->Student) : null;
```

Comment:
@nbey this seems like a kind of hacky space to do this, how are StudentTask records getting creating with StudentID set to people that aren't students? This check should be done during validation of StudentTask. We shouldn't be guarding against corrupt data on the UI/reporting layer